styles.css

```diff
@@ -0,0 +1,36 @@
+#main{
```

Comment:
Leaving a single white space between the selector and open curly brace would make your codebase clean and more readble. Doing so will leave no errors with the linters.

Reply:
Nicely done